core/src/components/item-sliding/item-sliding.tsx

```diff
@@ -462,15 +503,62 @@ export class ItemSliding implements ComponentInterface {
 
       openSlidingItem = undefined;
       style.transform = '';
+
+      if (animationType === 'modern') {
```

Comment:
Can we extract these keywords to consts or a type or something? When we eventually bike shed this will make potentially changing the names easier.